kinder/ci/workflows/patches-tasks.yaml

```diff
@@ -0,0 +1,200 @@
+vars:
+  # vars defines default values for variable used by tasks in this workflow;
+  # those values might be overridden when importing this files.
+  kubernetesVersion: v1.16.0
+  baseImage: kindest/base:v20190403-1ebf15f
+  image: kindest/node:test
+  clusterName: kinder-patches
+  kubeadmVerbosity: 6
+tasks:
+  - name: pull-base-image
+    description: |
+      pulls kindest/base image with docker in docker and all the prerequisites necessary for running kind(er)
+    cmd: docker
+    args:
+      - pull
+      - "{{ .vars.baseImage }}"
+  - name: add-kubernetes-versions
+    description: |
+      creates a node-image-variant by adding Kubernetes version "kubernetesVersion"
+      to be used when executing "kinder do kubeadm-init"
+    cmd: kinder
+    args:
+      - build
+      - node-image-variant
+      - --base-image={{ .vars.baseImage }}
+      - --image={{ .vars.image }}
+      - --with-init-artifacts={{ .vars.kubernetesVersion }}
+      - --with-upgrade-artifacts={{ .vars.kubernetesVersion }}
+      - --loglevel=debug
+    timeout: 10m
+  - name: create-cluster
+    description: |
+      create a set of nodes ready for hosting the Kubernetes cluster
+    cmd: kinder
+    args:
+      - create
+      - cluster
+      - --name={{ .vars.clusterName }}
+      - --image={{ .vars.image }}
+      - --control-plane-nodes=2
+      - --worker-nodes=1
+      - --loglevel=debug
+    timeout: 5m
+  - name: prepare patches
+    cmd: /bin/sh
+    args:
+      - -c
+      - |
+        mkdir -p /tmp/kubeadm-patches
+
+        cat <<EOF >/tmp/kubeadm-patches/kube-apiserver.yaml
+        {"metadata":{"annotations":{"patched":"true"}}}
+        EOF
+
+        cat <<EOF >/tmp/kubeadm-patches/kube-controller-manager+merge.yaml
+        metadata:
+          annotations:
+            patched: "true"
+        EOF
+
+        cat <<EOF >/tmp/kubeadm-patches/kube-scheduler0+strategic.yaml
+        metadata:
+          annotations:
+            patched: "true"
+        EOF
+
+        cat <<EOF >/tmp/kubeadm-patches/etcd+json.json
+        {"metadata":{"annotations":{"patched":"true"}}}
+        EOF
+  - name: prepare verify-patches.sh script
+    cmd: /bin/sh
+    args:
+      - -c
+      - |
+        cat <<EOF >/tmp/verify-patches.sh
+        #!/usr/bin/env bash
+        res=0
+        for d in /etc/kubernetes/manifests/*.yaml; do
+          if grep -q "patched: \"true\"" \$d ; then
+              echo "\$d is patched!"
+          else
+              echo "ERROR: \$d is not patched"
+              res=1
+          fi
+        done
+
+        if [[ "\${res}" = 0 ]]; then
+          echo "All verify checks passed, congrats!"
+          echo ""
+        else
+          echo "One or more verify checks failed! See output above..."
+          echo ""
+          exit 1
+        fi
+        EOF
+
+        chmod +x /tmp/verify-patches.sh
+  - name: copy verify-patches.sh on controlplane nodes
```

Comment:
As alternative to this approach, it is now possible to add those files at build time.
But I'm fine with both approaches